internal/cli/atlas/integrations/create/create.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package create
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func Builder() *cobra.Command {
+	const use = "create"
+	cmd := &cobra.Command{
+		Use:     use,
+		Aliases: []string{"update"},
```

Comment:
what happens if you call create multiple times? does it error?
Originally I defined this as create|update as the options are the same but you may need a different store for update

Reply:
I am using this endpoint https://docs.atlas.mongodb.com/reference/api/third-party-integration-settings-update/. If the user runs this command multiple times the service details will be updated all the time

<img width="824" alt="Screenshot 2020-09-03 at 17 42 22" src="https://user-images.githubusercontent.com/5663078/92136908-df348200-ee0c-11ea-9006-510679cb2752.png">

Reply:
and if doesn't exists will it create it?

Reply:
yes it does